docs/reference/builder.md

```diff
@@ -311,6 +311,12 @@ commands using a base image that does not contain `/bin/sh`.
 > If you want shell processing then either use the *shell* form or execute 
 > a shell directly, for example: `RUN [ "sh", "-c", "echo", "$HOME" ]`.
 
+> **Note**:
+> If you choose to use the *shell* form, any time you want to continue a single
```

Comment:
I believe this is true even for the exec form. e.g.:
```
RUN [  \
   'env' ]
```

It might be best to put this under the "Format" section near the top of this doc since this applies to some other Dockerfile commands too - e.g.:
```
ENV \
  foo1="hello"
```
not all though, which is kind of a bummer.